plugin-flex-ts-template-v2/src/feature-library/dispositions/custom-components/DispositionTab/DispositionTab.tsx

```diff
@@ -50,23 +72,45 @@ const DispositionTab = (props: OwnProps) => {
     if (isNotesEnabled() && notes) {
       payload.notes = notes;
     }
-
+    payload.custom_attributes = { ...customAttributes };
     dispatch(updateDisposition(payload));
   };
 
   const updateStoreDebounced = debounce(updateStore, 250, { maxWait: 1000 });
 
   useEffect(() => {
+    const getQueueFromInstantQuery = async (queueSid: string) => {
+      const queue = await FlexHelper.getQueue(queueSid);
+      setQueueName(queue?.queue_name || '');
+    };
+    getQueueFromInstantQuery(props.task?.queueSid || '');
+
     if (tasks && props.task && tasks[props.task.taskSid]) {
       if (tasks[props.task.taskSid].disposition) {
         setDisposition(tasks[props.task.taskSid].disposition);
       }
 
-      if (isNotesEnabled() && tasks[props.task.taskSid].notes) {
-        setNotes(tasks[props.task.taskSid].notes);
+      if (isNotesEnabled()) {
+        setNotes(tasks[props.task.taskSid].notes || '');
       }
+      // set custom attributes from Redux state
+      setCustomAttributes(tasks[props.task.taskSid].custom_attributes);
+      const options = {} as any;
```

Comment:
Would be great if you can define the type of the object rather than using `any` type.